spec/policies/organisation_policy_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe OrganisationPolicy do
+  subject { OrganisationPolicy.new(user, organisation) }
+
+  shared_examples 'not an organisation member' do
+    it { is_expected.not_to permit(:show)    }
+    it { is_expected.not_to permit(:create)  }
+    it { is_expected.not_to permit(:new)     }
+    it { is_expected.not_to permit(:update)  }
+    it { is_expected.not_to permit(:edit)    }
+    it { is_expected.not_to permit(:destroy) }
+  end
+
+  shared_examples 'an organisation member' do
+    let(:organisations) { FactoryBot.create(:organisations, organisation_ids: 1) }
```

Comment:
i don't think organisations have an attribute called "organisations_ids".

Are you trying to create a list of organisations?